github.md

```diff
@@ -10,23 +18,33 @@ Branch protection should be enabled for the default/release branch of a reposito
 - Require branches to be up to date before merging
 - Include administrators
 
-## Codeowners
+### Access
+- Access should be granted to [GitHub teams][gh-teams]
+  - Should an individual leave the GitHub organisation, they'll automatically lose access to any repository in the organisation
+- Avoid individual access
+  - Should the individual leave the GitHub organisation, they'll retain access until manually removed
+- Open the repository to the department
+  - [`@guardian/guardian-developers-read`][gh-read] should have read access
+  - [`@guardian/guardian-developers-write`][gh-write] should have write access
+
+### Codeowners
 
-It is recommended to include a `CODEOWNERS` file which references the GitHub team or teams that are responsible for the repository. Where this file is present, it is also recommended to enable the `Require review from Code Owners` option in branch protection.
+Include a [`CODEOWNERS`][gh-codeowners] file which references the GitHub team(s) responsible for the repository.
```

Comment:
I'd be tempted to indicate why this is important, because I feel like it is an easy one to ignore for repos where ownership is a bit ambiguous (but that's where it's most needed IMO).

(Feel free to reword, just for example) `On the one hand, this is helpful from a practical point of view, for the team responsible team to receive notifications of contributions and control changes.  On top of that it is also important at the org level to identify owners for every repository so that we can better ensure security and maintenance work is under the remit of a team for all our (production) code.`

Reply:
Great suggestion. I wonder if the doc could be restructured for this reason too. Something like:

```
# Topic
**Recommendation:**
Something short and to the point.

**Reason:**
A paragraph explaining why it's a good thing.
```

As a way of surfacing the recommendation really clearly.

Reply:
^ this made the doc very busy! I've added an explanation for CODEOWNERS now.